1.5/Mods/Biotech/Defs/aotcGeneProcessors.xml

```diff
@@ -0,0 +1,211 @@
+<Defs>
+  <ThingDef ParentName="BuildingBase">
```

Comment:
When creating something requiring a DLC, don't forget the tag "MayRequire="[Name of the mod]""

Here, I think the line should look like   <ThingDef ParentName="BuildingBase"  MayRequire="Ludeon.Rimworld.Biotech">